icons/fish.svg

```diff
@@ -0,0 +1,18 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+ <path d="M7 12c.94-3.465 4.442-6 8-6s6.06 2.535 7 6c-.94 3.466-3.442 6-7 6s-7.06-2.534-8-6Z"/>
+ <path d="M18 12v.5"/>
+ <path d="M16 6a8.565 8.565 0 0 0 0 11"/>
+ <path d="M10.5 6.5c-.134-1.092-.503-1.788-1.221-2.64-.283-.335-.057-.86.381-.86h4.105a2 2 0 0 1 1.98 1.717l.17 1.19"/>
+ <path d="M7.9 9.5c-.6-2.2-2.105-3.534-4.921-4-.214 2.556-.03 3.98 1.007 6.5-1.036 2.52-1.22 3.944-1.007 6.5 2.816-.466 4.32-1.8 4.921-4"/>
+ <path d="M11.5 17.5c-.18.828-.316 1.706-.81 2.571-.216.381.032.929.47.929h2.105a2 2 0 0 0 1.98-1.717L15.43 18"/>
```

Comment:
Cute, much nicer than what I had lying around as a draft. :D 

There's a minor guideline violation when it comes to the tail meeting the body:
![image](https://user-images.githubusercontent.com/17746067/199508315-b6f09813-135b-46c2-8626-ba01f448ddbf.png)

Since it looks pretty fine on 100%, I'm personally OK with letting this one slide as long as @ericfennis is OK with it.
![fish](https://user-images.githubusercontent.com/17746067/199508499-7b49b414-6f69-4781-8a70-9571f76ec82b.png)

I would however consider making the fins pointed, as illustrated above.